packages/graphql-testing/src/links/mocks.ts

```diff
@@ -7,7 +7,7 @@ export class MockLink extends ApolloLink {
     super();
   }
 
-  request(operation: Operation) {
+  request(operation: Operation): Observable<any> {
```

Comment:
is `any` right here?

Reply:
I spent a bit of time trying to address these. The changes were significant in some cases. I'd prefer having those changes in a subsequent PR if that's ok with you.